rchie01/dotTheory-React
# RCT-102: Using Create React App

# Related Tickets

https://codeandtheory.atlassian.net/browse/RCT-130

# Description

- Creates a simple React Application using `create-react-app`
- This PR intends to demonstrate the use of `create-react-app`

# Validation


- Clone the repository using SSH or HTTPS
- Open the search-app folder located in the RCT-102 folder.
- Run `npm start`

The user should be able to see components imported from `react-bootstrap`.

# Risks
- N/A
